app/promptfoo/promptfooconfig.ci.yaml

```diff
@@ -26,6 +26,8 @@ defaultTest:
 evaluateOptions:
   delay: 1000
   showProgressBar: true
+
+commandLineOptions:
```

Comment:
Based on https://www.promptfoo.dev/docs/usage/command-line/#specifying-command-line-options-in-config